conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/model/workspace/Workspace.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.conversation.v1.model.workspace;
```

Comment:
I was trying to follow the rest of WDC's paradigm when I was updating this and it seemed that by adding everything to model, it was become a cluttered and confusing package. For example, there would then be two types of Intent objects, one for message response and one for the actual intent. I thought organizing by set of endpoints, such as they way the API documentation was laid out made sense.

Do you have a suggested organization scheme that I can follow?